src/core/reducers/abuse.js

```diff
@@ -157,6 +196,7 @@ export default function abuseReducer(
         loading: false,
       };
     }
+    case INITIATE_ADDON_ABUSE_REPORT_VIA_FIREFOX:
     case SEND_ADDON_ABUSE_REPORT:
```

Comment:
It seems a bit weird to me that we are handling those two actions the same way given that their intents are very different.

Reply:
In a way their intents are identical, but the original one is named in a way that doesn't make that obvious. Both actions are about initiating an attempt to report abuse in an async manner. The old one uses an AMO API call, and the new one uses a built-in Firefox API. In both cases the reducer is intended to update the UI to indicate that an abuse report is in progress, but the difference is that each triggers a different saga to accomplish that, which is why there are still two differently named actions. So to me it all makes sense. 😄